web/src/hotkey.js

```diff
@@ -975,7 +975,10 @@ export function process_hotkey(e, hotkey) {
             popovers.show_sender_info();
             return true;
         case "toggle_reactions_popover": // ':': open reactions to message
-            reactions.open_reactions_popover();
+            emoji_picker.toggle_emoji_popover(
+                reactions.get_reactions_popver_target(),
+                message_lists.current.selected_id(),
+            );
```

Comment:
I feel like this creates a function with a pretty poor interface; `reactions.get_reactions_popver_target()` accesses `message_lists.current.selected_id` in several forms multiple times, and we're doing it here in the caller as well. 

I also note that we fetched `const msg = message_lists.current.selected_message();` earlier in this function, just above this block ...